### PR TITLE
Update DG, PPP, About Us. Clean Code

### PR DESCRIPTION
**Pull request by ParthGandhiNUS**
As title says
Also added a picture of myself

### CODE REVIEW

File: src/main/java/classify/commands/ListStudentsCommand.java [hunk 1]
```diff
@@ -70,11 +70,11 @@ public static void listStudentsBySubject(ArrayList<Student> masterStudentList, S
                 } else {
                     UI.println("- " + student.getName() + " - " + "No Classes Attended found for " + subject);
                 }
-                found = true;
+                isFound = true;
```
Comment (Cryolian) [inline src/main/java/classify/commands/ListStudentsCommand.java:73]: good change!

File: docs/team/Parth Image.JPG
Comment (Cryolian) [inline docs/team/Parth Image.JPG:1]: hensum

**Review by Cryolian — APPROVED** [review]
looks good to me

File: docs/team/Parth Image.JPG
Comment (ParthGandhiNUS) [reply docs/team/Parth Image.JPG:1]: Thank you sir!

**Pull request merged**